sqlmesh/core/config/connection.py

```diff
@@ -381,14 +381,20 @@ def _static_connection_kwargs(self) -> t.Dict[str, t.Any]:
         """kwargs that are for execution config only"""
         from sqlmesh import __version__
 
-        custom_user_agent_config = {"custom_user_agent": f"SQLMesh/{__version__}"}
         connection_str = "md:"
+
+        query_params = []
```

Comment:
you can use urlencode

from urllib import urlencode

urlencode(...) 

instead of manually doing string concat